mllib/src/test/scala/org/apache/spark/ml/regression/DecisionTreeRegressorSuite.scala

```diff
@@ -96,6 +108,15 @@ class DecisionTreeRegressorSuite
       assert(variance === expectedVariance,
         s"Expected variance $expectedVariance but got $variance.")
     }
+
+    val toyDF = TreeTests.setMetadata(toyData, Map.empty[Int, Int], 0)
+    dt.setMaxDepth(1)
+      .setMaxBins(6)
```

Comment:
I'd like to remove the explicit setting since the default value(32) meets your needs. We want to make Jenkins logging clean and reduce the number of warnings if possible.